### DISCUSSION TITLE
Fastest way to load .prg from host and execute in VirtualC64?

### DISCUSSION THREAD

**Discussion by boeckmann** [dirkwhoffmann/virtualc64#739] [Q&A] [answered]
Hello, I am searching for the fastest way to load a .prg file from the host system into VirtualC64 memory and execute it. I am able to load the .prg file and run it via the open menu entry.

Is there any "reload" command to reload the previously selected .prg file from the host again and reexecute the binary without going through the open menu and creating a new document? That would speed up the development cycle significantly.

**Comment by dirkwhoffmann** [comment, marked as answer]
I usually drag the PRG into the emulator window and hit reset. This is the fastest way to get the PRG in.

**Comment by boeckmann** [comment]
Danke für den Tipp!

**Comment by roaddan** [comment]
Hi Is there a way to load two disk images one on each drive using the command line. I am testing program that may need two disquette or I would like to load a utility disk on drive 9 with my program on drive 8. In vice we van use -8 filename -9 filename. Something like that would be nice. Great application by the way.
